documentation/api/io.strimzi.api.kafka.model.AclRule.adoc

```diff
@@ -1,3 +1,5 @@
+xref:type-AclRule-schema-{context}[Full list of `AclRule` schema properties]
```

Comment:
I wonder if instead of adding these manually, we should generate these in the generator. Otherwise we might soon add new modules and forget to add this. Since these are just includes, it should be easy to generate it before it It could be just added before the line 419 in `DocGenerator`.

Reply:
Yeah. I thought about this yesterday. Not sure why I went for the manual in the end.